packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountWelcomeScreen.js

```diff
@@ -1,156 +1,164 @@
-              label={s.strings.exit}
-            />
+  return (
+    <ThemedScene>
+      <SimpleSceneHeader>{s.strings.get_started}</SimpleSceneHeader>
+      <View style={styles.content}>
+        <Image source={LOGO_BIG} style={styles.logo} resizeMode="contain" />
+        <EdgeText style={styles.welcome}>
+          {sprintf(s.strings.welcome, appName)}
+        </EdgeText>
+        <View style={styles.advantage}>
+          <Image
+            source={WELCOME_LOCK}
+            style={styles.advantageImage}
+            resizeMode="contain"
+          />
+          <View style={styles.advantageTextContainer}>
+            <EdgeText style={styles.advantageTitle}>
+              {sprintf(s.strings.welcome_advantage_one_title, appName)}
+            </EdgeText>
+            <EdgeText style={styles.advantageDescription}>
+              {s.strings.welcome_advantage_one_description_1}
+            </EdgeText>
+            <EdgeText style={styles.advantageDescription}>
+              {s.strings.welcome_advantage_one_description_2}
+            </EdgeText>
           </View>
-          <View style={styles.row2}>
-            <ImageHeaderComponent src={Assets.WELCOME} />
-          </View>
-          <View style={styles.row3}>
-            <T style={styles.instructionsText}>
-              {sprintf(
-                s.strings.welcome_one,
-                this.props.branding.appName || s.strings.app_name_default
-              )}
-            </T>
-          </View>
-          <View style={styles.row4} />
-          <View style={styles.row5}>
-            <T style={styles.callToAction}>{s.strings.start_username}</T>
-          </View>
-          <View style={styles.row6}>
-            <Button
-              testID="getStartedButton"
-              onPress={this.props.onDone}
-              downStyle={styles.nextButton.downStyle}
-              downTextStyle={styles.nextButton.downTextStyle}
-              upStyle={styles.nextButton.upStyle}
-              upTextStyle={styles.nextButton.upTextStyle}
-              label={s.strings.get_started}
-            />
+        </View>
+        <View style={styles.divider} />
+        <View style={styles.advantage}>
+          <Image
+            source={WELCOME_SHIELD_KEY}
+            style={styles.advantageImage}
+            resizeMode="contain"
+          />
+          <View style={styles.advantageTextContainer}>
+            <EdgeText style={styles.advantageTitle}>
+              {s.strings.welcome_advantage_two_title}
+            </EdgeText>
+            <EdgeText style={styles.advantageDescription}>
+              {sprintf(s.strings.welcome_advantage_two_description_1, appName)}
+            </EdgeText>
+            <EdgeText style={styles.advantageDescription}>
+              {s.strings.welcome_advantage_two_description_2}
+            </EdgeText>
+            <EdgeText style={styles.advantageDescription}>
+              {s.strings.welcome_advantage_two_description_3}
+            </EdgeText>
```

Comment:
This three lines shouldn't be 3 separate text component.
It can look really off on different size screens or different translations.

They should be joined into one sentence with auto wrapping.